tests/integration-tests/configs/common/rocky8.yaml

```diff
@@ -0,0 +1,578 @@
+# FIXME: Remove this YAML once we officially release Rocky8 AMI
+# This duplicates some sections of common.yaml to include Rocky8 AMI in daily Integ tests
```

Comment:
In future we can think to do it more generic, calling it `not_released_oses.yaml` and define a new variable like `NOT_RELEASED_OSES`
```
{%- set NOT_RELEASED_OSES = ["rocky8"] -%}
```